project/src/components/map/map.tsx

```diff
@@ -8,7 +8,7 @@ import { connect, ConnectedProps } from 'react-redux';
 
 type MapProps = {
   offers: Offer[];
-  activeOffer: Offer | null;
+  activeOffer?: Offer | null;
```

Comment:
```suggestion
  activeOffer: Offer | null;
```
передавай где надо null